src/API/Reports/Coupons/DataStore.php

```diff
@@ -347,14 +380,16 @@ public static function sync_order_coupons( $order_id ) {
 		$coupon_items       = $order->get_items( 'coupon' );
 		$coupon_items_count = count( $coupon_items );
```

Comment:
It doesn't look like this is used - in fact it doesn't look like it was really used at all

Reply:
It's used here: https://github.com/woocommerce/woocommerce-admin/pull/4671/files/e3e0583296e35f6805c92034014be1b8b23018df#diff-a16bed59f4a4360d1c1fa341022c0adcR437